k2/csrc/fsa_utils.cu

```diff
@@ -240,16 +238,15 @@ static Fsa TransducerFromStream(std::string first_line, std::istringstream &is,
     K2_CHECK_EQ(openfst, true);
     K2_CHECK_EQ(original_final_states.size(), original_final_weights.size());
     int32_t super_final_state = max_state + 1;
-    for (std::size_t = 0; i != original_final_states.size(); ++i) {
-      arcs.emplace_back(original_final_states[i],
-                        super_final_state,
-                        -1,             // kFinalSymbol
+    for (std::size_t i = 0; i != original_final_states.size(); ++i) {
```

Comment:
@chenguoguo 
FYI: we use `clang-format==9.0.0` for auto formatting.

Reply:
Thanks. Is that something that can be enforced? E.g., as part of the CI style checking?